go.sum

```diff
@@ -87,20 +87,55 @@ github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03
 github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802/go.mod h1:IVnqGOEym/WlBOVXweHU+Q+/VP0lqqI8lqeDx9IjBqo=
 github.com/HdrHistogram/hdrhistogram-go v1.0.1 h1:GX8GAYDuhlFQnI2fRDHQhTlkHMz8bEn0jTI6LJU0mpw=
 github.com/HdrHistogram/hdrhistogram-go v1.0.1/go.mod h1:BWJ+nMSHY3L41Zj7CA3uXnloDp7xxV0YvstAE7nKTaM=
+github.com/Knetic/govaluate v3.0.1-0.20171022003610-9aa49832a739+incompatible/go.mod h1:r7JcOSlj0wfOMncg0iLm8Leh48TZaKVeNIfJntJ2wa0=
```

Comment:
I guess we didn't tidy this in a long time?

Reply:
Ah, I found it. I have Go 1.18 locally and the `go` tool has changed a bit:

>The go mod tidy command now retains additional checksums in the go.sum file for modules whose source code is needed to verify that each imported package is provided by only one module in the [build list](https://tip.golang.org/ref/mod#glos-build-list). Because this condition is rare and failure to apply it results in a build error, this change is not conditioned on the go version in the main module's go.mod file.

Reply:
(Though I cannot explain what's different about branch/v8 since we didn't see as many new sums in v7 or v9)

Reply:
Ok, in that case I think it's fine.